Add cups information for CLS (#307)
cap-js/telemetry: README.md

}
```

> If you are using user-provided binding for Cloud Logging, make sure that the instance has a tag "cloud-logging":
> ```bash
> cf cups <your-service-name> -p ups.json -t "cloud-logging"
> ```

Reviewer: ```suggestion
If you are binding your app to SAP Cloud Logging via a [user-provided service instance](https://docs.cloudfoundry.org/devguide/services/user-provided.html), make sure that it has a tag `cloud-logging`.
```
